docs/notifications.json

```diff
@@ -1,19 +1,4 @@
 [
-  {
-    "id": 35,
-    "title": "Let's translate!",
-    "text": "<a style=\"color: inherit;\" target=\"_blank\" rel=\"noopener\" data-ga-event-category=\"l10n\" data-ga-event-action=\"notification\" data-ga-event-label=\"zh\" href=\"https://translate.mui.com/\">帮助 MUI 将文档翻译成中文</a>. 🇨🇳",
-    "userLanguage": "zh"
-  },
-  {
-    "id": 53,
```

Comment:
Note that this one and the previous one are meant it be ever green notifications. The first one is only shown to Chinese users. The second one correlates with x2 more followers the last time we tried to measure the impact https://github.com/mui/material-ui/pull/20337